bundles/org.openhab.binding.netatmo/src/main/resources/ESH-INF/config/config.xml

```diff
@@ -61,7 +61,7 @@
 			<advanced>true</advanced>
 		</parameter>
 
-		<parameter name="reconnectInterval" type="integer" required="true">
+		<parameter name="reconnectInterval" type="integer" unit="s" required="true">
 			<label>Reconnect Interval</label>
 			<description>The reconnection interval to Netatmo API (in s).</description>
```

Comment:
I know that some UIs (e.g. Paper UI) - which will be removed in OH3 - do not show this "unit" but we prefer to not have it inside the description.

Reply:
Removing the unit from the description would have an immediate negative impact on user experience through one of the primary UIs users use to interact with openHAB. Until we we can replace Paper UI the units should be left in the description. We can always go back and remove units from the description later.